leapsecs_read.c

```diff
@@ -0,0 +1,49 @@
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <errno.h>
+extern int errno;
+#include "leapsecs.h"
+
+struct tai *leapsecs = 0;
+int leapsecs_num = 0;
+
+int leapsecs_read()
+{
+  int fd;
+  struct stat st;
+  struct tai *t;
+  int n;
+  int i;
+  struct tai u;
+
+  fd = open("/etc/leapsecs.dat",O_RDONLY | O_NDELAY);
```

Comment:
Adding this new file as a resource, I would be useful to have a documentation, maybe on the wiki to signal this change of behavior.  Containers using qmail would, for instance, lack the file, or the system may simply not have the tai file installed.

Does this mean we need to ship the tools to maintain this `/etc/leapsecs.dat` or do we let libtai do that work?

Hopefully I can come-up with something simple that converts this https://hpiers.obspm.fr/iers/bul/bulc/ntp/leap-seconds.list to the libtai format.